src/pcs/c1/PageMods.js

```diff
@@ -0,0 +1,145 @@
+import BodySpacingTransform from '../../transform/BodySpacingTransform'
+import CollapseTable from '../../transform/CollapseTable'
+import DimImagesTransform from '../../transform/DimImagesTransform'
+import LazyLoadTransformer from '../../transform/LazyLoadTransformer'
+import PlatformTransform from '../../transform/PlatformTransform'
+import Scroller from './Scroller'
+import ThemeTransform from '../../transform/ThemeTransform'
+
+/**
+ * Executes common JS functionality the client should start, like hooking up events for
+ * lazy loading and table collapsing/expanding.
+ * Client side complement of server-side DOM transformations.
+ * Note: this will be called automatically when DOM is ready
+ * @param {!Window} window
+ * @param {!Document} document
+ * @return {void}
+ */
+const onPageLoad = (window, document) => {
+  const lazyLoader = new LazyLoadTransformer(window, 2)
+  lazyLoader.collectExistingPlaceholders(document.body)
+  lazyLoader.loadPlaceholders()
+
+  CollapseTable.setupEventHandling(window, document, true, Scroller.scrollWithDecorOffset)
+}
+
+/**
+ * @typedef {function} OnSuccess
+ * @return {void}
+ */
```

Comment:
I thought this was out of place at first, but I see it's defining a type used for callbacks below.  Still, it seems like overkill to define a type for callback functions, and more work for the reader — maybe better to just declare the callback params as plain old Functions.

Reply:
I saw this in other page lib modules. Have to use something like Foo~Function to make eslint happy.

Reply:
OK.  Now I see the pattern used elsewhere in this codebase; not sure why I missed it the other day.